pkg/sdk/go/channels_test.go

```diff
@@ -67,6 +71,20 @@ func TestCreateChannel(t *testing.T) {
 			err:     nil,
 			empty:   false,
 		},
+		{
+			desc:   "create a new channel with external UUID",
+			channel: channel1,
```

Comment:
To be consistent with other naming this should be `thExtID`

Reply:
Done